xla/translate/mhlo_to_hlo/mlir_hlo_to_hlo.cc

```diff
@@ -834,10 +834,19 @@ bool SimplyReturnedOp(mlir::Operation* op) {
 namespace mlir {
 namespace mhlo {
 namespace {
+LogicalResult ExportXlaOp(CollectiveBroadcastOp op, OpLoweringContext ctx) {
```

Comment:
Can you add a filecheck test to the following files, just to ensure round-tripping doesn't break:

- https://github.com/openxla/xla/blob/d239bd4230482fc02fda58bb8ac3bde3a3339c0c/xla/translate/mhlo_to_hlo/tests/export.mlir#L589
- https://github.com/openxla/xla/blob/4d7e7b96776aff48695c9e4ca64f666a5e9d5fb5/xla/translate/hlo_to_mhlo/tests/import.hlotxt#L266